.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,8 @@
 blank_issues_enabled: true
 contact_links:
+  - name: Winget Client issues
```

Comment:
nit: consistency
```suggestion
  - name: WinGet Client issues
```

Reply:
Sadly, this should be "Windows Package Manager Client Issues".

Reply:
Nevermind. "winget client" is fine. We just shouldn't use "winget" without specifying client. Otherwise, we should use the full name.